requirements.txt

```diff
@@ -15,7 +15,7 @@ chardet==3.0.4
 click==7.1.2
 cognitojwt==1.2.2
 coverage==5.4
-cryptography==3.3.2
+cryptography==3.3.1
```

Comment:
What's the intention behind downgrading the cryptography library? This was recently upgraded due to a security vulnerability.

Reply:
Good to know. I'm assuming it's because the undetected_chrome package dependency relies on that version. I'll upgrade it and test.

Reply:
Awesome. For your reference and so you know I'm not crazy, here was the automated pull request for it a couple days ago. https://github.com/cisagov/domain-manager-api/pull/159

Reply:
No worries! Good catch